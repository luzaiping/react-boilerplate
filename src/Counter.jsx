import React from 'react';

// eslint-disable-next-line react/prop-types
const Counter = ({ count }) => (
  <div>
    <p>{count}</p>
  </div>
);

export default Counter;
