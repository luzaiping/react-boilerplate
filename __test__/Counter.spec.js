import React from 'react';
import renderer from 'react-test-renderer';

import Counter from '../src/Counter';

describe('Counter', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<Counter count={0} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
