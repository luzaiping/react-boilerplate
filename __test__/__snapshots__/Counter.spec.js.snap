// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter snapshot renders 1`] = `
<div>
  <p>
    0
  </p>
</div>
`;
